skll/experiments.py

```diff
@@ -897,8 +894,12 @@ def _create_learner_result_dicts(task_results,
                     f_sum_dict[actual_label] += float(label_f)
                 result_row = ([actual_label] + conf_matrix[i] +
                               [label_prec, label_recall, label_f])
-                result_table.add_row(result_row)
+                rows.append(result_row)
 
+            result_table = tabulate(rows, 
+                                    headers=headers, 
+                                    floatfmt=".3f", 
+                                    tablefmt="grid",)
```

Comment:
- Please remove the extra whitespace after the commas.
- Add the keyword argument `stralign="right"` after `headers` to match the right alignment specification from PrettyTable.